### PR TITLE
Sentry integration

### PR DESCRIPTION
**Pull request by edigaryev**
See https://github.com/cirruslabs/cirrus-cli/issues/587.

### CODE REVIEW

File: cmd/agent/main.go [hunk 1]
```diff
@@ -75,6 +76,57 @@ func main() {
 		"working directory to use when spawned via Persistent Worker")
 	flag.Parse()
 
+	// Initialize Sentry
+	err := sentry.Init(sentry.ClientOptions{
```
Comment (fkorotkov) [inline cmd/agent/main.go:80]: Do we need to set release similar to how softnet and tart does?

Comment (edigaryev) [reply cmd/agent/main.go:86]: See https://github.com/cirruslabs/cirrus-ci-agent/pull/264/commits/e1a0fa1040ca00465b2c25d1e72afcf4743a29f4.

File: cmd/agent/main.go [hunk 2]
```diff
@@ -75,6 +76,64 @@ func main() {
...
 
+	// Initialize Sentry
+	var release string
+
+	if version != "unknown" {
+		release = version
+	}
+
+	err := sentry.Init(sentry.ClientOptions{
+		Release:          release,
```
Comment (fkorotkov) [inline cmd/agent/main.go:87]:
```suggestion
		Release:          fmt.Sprintf("cirrus-ci-agent@%s", release),
```

Comment (edigaryev) [reply cmd/agent/main.go:87]: Good catch, see https://github.com/cirruslabs/cirrus-ci-agent/pull/264/commits/44507df07eb46c6a7054a2f9af60cd20358cc38e for a fix.

**Review by fkorotkov — APPROVED** [review]

**Pull request merged**